packages/link/src/utils/types.ts

```diff
@@ -127,6 +135,18 @@ export interface LinkOptions {
    * Can be used to initialize the crypto transfers flow as an alternative to the target addresses.
    */
   transferDestinationTokens?: IntegrationAccessToken[]
+
+  /**
+   * (Optional) An array of Wagmi injected connector data. React hook
+   * It should send this list directly to our iframe.
+   */
+  injectedConnectors?: WagmiInjectedConnectorData[]
```

Comment:
Please remove these parameters. The client's application does not need to provide us the available connectors.